### PR TITLE
Prepare repository for VAST v1.1.0-rc1

### PR DESCRIPTION
**Pull request by dominiklohmann**
The usual.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,46 @@ This file is generated automatically. Add individual changelog entries to the 'c
 
 This changelog documents all notable changes to VAST and is updated on every release. Changes made since the last release are in the [changelog/unreleased directory][unreleased].
 
+## [v1.1.0-rc1]
+
+### :zap: Breaking Changes
+
+- VAST has a new transform API: the `apply()` function is changed to `add()` and `finish()`. The `add()` function receives arrow batches instead of `table_slice`s. Multiple calls to the `add()` function adds multiple batches to the transformation, and the `finish()` function returns the transformed slices; this enables the transform to change multiple table slices simultaneously.
+  [#2020](https://github.com/tenzir/vast/pull/2020)
```
Comment (dominiklohmann) [inline CHANGELOG.md:14]: This isn't really user-facing; let's remove it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,46 @@ This file is generated automatically. Add individual changelog entries to the 'c
...
+- The built-in 'select' transform step may now use filter expressions that match *all* events of the input.
+  [#2064](https://github.com/tenzir/vast/pull/2064)
+
+- VAST has a new `query_language` plugin type that allows for adding additional query language frontends. The plugin performs one function: compile a string into a VAST expression. The new `sigma` plugin demonstrates usage of this plugin type.
+  [#2074](https://github.com/tenzir/vast/pull/2074)
+
+- The new builtin `rename` transform step allows for renaming event types as part of a transformation. This is especially useful when you want to ensure that a repeatedly triggered transformation does not affect already transformed events.
+  [#2076](https://github.com/tenzir/vast/pull/2076)
+
+- The new `aggregate` transform plugin allows for flexibly grouping and aggregating events. We recommend using it alongside the [`compaction` plugin](https://docs.tenzir.com/vast/features/compaction), e.g., for compaction `suricata.flow` or `zeek.conn` events after a certain amount of time in the database.
```
Comment (dominiklohmann) [inline CHANGELOG.md:38]:
```suggestion
- The new `aggregate` transform plugin allows for flexibly grouping and aggregating events. We recommend using it alongside the [`compaction` plugin](https://docs.tenzir.com/vast/features/compaction), e.g., for compacting `suricata.flow` or `zeek.conn` events after a certain amount of time in the database.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,46 @@ This file is generated automatically. Add individual changelog entries to the 'c
...
+  [#2076](https://github.com/tenzir/vast/pull/2076)
+
+### :beetle: Bug Fixes
+
+- A long-standing performance bug that caused query evaluation to look at more candidate partitions than necessary no longer exists: A query for `ts`, e.g., for matching the field `zeek.conn.ts`, stops evaluating unrelated fields that end in `ts`, e.g., `suricata.flow.flow.pkts` at an earlier point in the pipeline. Similarly, a field exactly named `ts` of a type incompatible with the query will no longer lead to the partition being considered.
+  [#2086](https://github.com/tenzir/vast/pull/2086)
```
Comment (dominiklohmann) [inline CHANGELOG.md:44]: This has way too much detail, users don't need so many details here.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,41 @@ This file is generated automatically. Add individual changelog entries to the 'c
...
+
+- The experimental aging feature is now deprecated. If you require this feature, please contact us about the [compaction plugin](https://docs.tenzir.com/vast/features/compaction) which is its more capable replacement.
+  [#2087](https://github.com/tenzir/vast/pull/2087)
+
+### :gift: Features
+
+- The built-in 'select' transform step may now use filter expressions that match *all* events of the input.
+  [#2064](https://github.com/tenzir/vast/pull/2064)
+
+- VAST has a new `query_language` plugin type that allows for adding additional query language frontends. The plugin performs one function: compile a string into a VAST expression. The new `sigma` plugin demonstrates usage of this plugin type.
```
Comment (mavam) [inline CHANGELOG.md:27]:
```suggestion
- VAST has a new *query language plugin* that allows for adding additional query language frontends. The plugin performs one function: compile user input into a VAST expression. The new `sigma` plugin demonstrates usage of this plugin type.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,41 @@ This file is generated automatically. Add individual changelog entries to the 'c
...
+  [#2074](https://github.com/tenzir/vast/pull/2074)
+
+- The new built-in `rename` transform step allows for renaming event types as part of a transformation. This is especially useful when you want to ensure that a repeatedly triggered transformation does not affect already transformed events.
+  [#2076](https://github.com/tenzir/vast/pull/2076)
+
+- The new `aggregate` transform plugin allows for flexibly grouping and aggregating events. We recommend using it alongside the [`compaction` plugin](https://docs.tenzir.com/vast/features/compaction), e.g., for compacting `suricata.flow` or `zeek.conn` events after a certain amount of time in the database.
```
Comment (mavam) [inline CHANGELOG.md:33]:
```suggestion
- The new `aggregate` transform plugin allows for flexibly grouping and aggregating events. We recommend using it alongside the [`compaction` plugin](https://docs.tenzir.com/vast/features/compaction), e.g., for rolling up events into a more space-efficient representation after a certain amount of time.
```

**Review by mavam — COMMENTED** [review]
The usual wordsmithing.

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,6 +6,41 @@ This file is generated automatically. Add individual changelog entries to the 'c
...
+
+- The `msgpack` encoding option is now deprecated. VAST issues a warning on startup and automatically uses the `arrow` encoding instead. A future version of VAST will remove this option entirely.
+  [#2087](https://github.com/tenzir/vast/pull/2087)
+
+- The experimental aging feature is now deprecated. If you require this feature, please contact us about the [compaction plugin](https://docs.tenzir.com/vast/features/compaction) which is its more capable replacement.
+  [#2087](https://github.com/tenzir/vast/pull/2087)
+
+### :gift: Features
+
+- The built-in 'select' transform step may now use filter expressions that match *all* events of the input.
```
[the remark this answers is not in the document]
Comment (dominiklohmann) [reply CHANGELOG.md:24]: I think the changelog entry is actually wrong, the bug fix for project and select was that we did not handle transformations correctly that dropped all columns and rows respectively.

File: CHANGELOG.md (hunk 4 above)
Comment (dominiklohmann) [reply CHANGELOG.md:27]: This is wrong. There's an abstraction for query language plugins, but that does not mean that there is a query language plugin.

File: CHANGELOG.md [hunk 7]
```diff
@@ -6,6 +6,41 @@ This file is generated automatically. Add individual changelog entries to the 'c
...
+
+- The new built-in `rename` transform step allows for renaming event types as part of a transformation. This is especially useful when you want to ensure that a repeatedly triggered transformation does not affect already transformed events.
+  [#2076](https://github.com/tenzir/vast/pull/2076)
+
+- The new `aggregate` transform plugin allows for flexibly grouping and aggregating events. We recommend using it alongside the [`compaction` plugin](https://docs.tenzir.com/vast/features/compaction), e.g., for compacting `suricata.flow` or `zeek.conn` events after a certain amount of time in the database.
+  [#2076](https://github.com/tenzir/vast/pull/2076)
+
+### :beetle: Bug Fixes
+
+- A long-standing performance bug in the first stage of query evaluation caused VAST to possibly consider candidate partitions for the second stage of query evaluation for fields with matching name suffixes, but mismatching types. This bug no longer exists, and should result in a considerable speedup of some queries.
```
[the remark this answers is not in the document]
Comment (dominiklohmann) [reply CHANGELOG.md:38]: Your rewrite dropped the information about mismatching types.

File: CHANGELOG.md [hunk 8]
```diff
@@ -6,6 +6,41 @@ This file is generated automatically. Add individual changelog entries to the 'c
...
+
+### :warning: Changes
+
+- VAST no longer attempts to intepret query expressions as Sigma rules automatically. Instead, this functionality moved to a dedicated `sigma` query language plugin that must explicitly enabled at build time.
+  [#2074](https://github.com/tenzir/vast/pull/2074)
+
+- The `msgpack` encoding option is now deprecated. VAST issues a warning on startup and automatically uses the `arrow` encoding instead. A future version of VAST will remove this option entirely.
+  [#2087](https://github.com/tenzir/vast/pull/2087)
+
+- The experimental aging feature is now deprecated. If you require this feature, please contact us about the [compaction plugin](https://docs.tenzir.com/vast/features/compaction) which is its more capable replacement.
```
[the remark this answers is not in the document]
Comment (dominiklohmann) [reply CHANGELOG.md:19]: I took the remaining wording suggestion, but used aging over ageing.

File: CHANGELOG.md (hunk 4 above)
Comment (dominiklohmann) [reply CHANGELOG.md:27]: I took your suggestion, but left in the word type.

File: CHANGELOG.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (dominiklohmann) [reply CHANGELOG.md:38]: I rewrote your suggestion slightly to include that information again.

File: CHANGELOG.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (mavam) [reply CHANGELOG.md]: You're right, eagle eye!

File: CHANGELOG.md (hunk 4 above)
Comment (mavam) [reply CHANGELOG.md]: Makes sense, actually.

**Review by mavam — APPROVED** [review]

**Pull request merged**